### DISCUSSION TITLE
How to stash part of change?

### DISCUSSION THREAD

**Discussion by lixiaoquan** [jesseduffield/lazygit#1332] [Q&A] [answered]
How to achieve what 'git stash -p' does?

Scenario: Sometimes I made some changes in a few files and part of it broke something, and to find out the culprit, I'd like to disable/enable part of changes quickly.

**Comment by jesseduffield** [comment, marked as answer]
The way I do it is to select the files that you want to stash, or even individual lines in files that you want to stash by hitting enter on the file and then using the line-by-line staging mode. Then I press shift+S and select to stash staged files
